src/qililab/instruments/quantum_machines/quantum_machines_cluster.py

```diff
@@ -401,6 +402,8 @@ def _get_elements_and_mixers_config(self) -> tuple:
     _config: DictQuaConfig
     _octave_config: QmOctaveConfig | None = None
     _is_connected_to_qm: bool = False
+    _intermediate_frequency: dict[str, float] = {}
```

Comment:
Renaming suggestion for more clarity of the atrribute usage.
```suggestion
    _pending_set_intermediate_frequency: dict[str, float] = {}
```